lib/services/keyVaultManagement/lib/keyVaultManagementClient.js

```diff
@@ -537,7 +537,7 @@ var VaultOperations = ( /** @lends VaultOperations */ function() {
     url2 = url2 + '/vaults/';
     url2 = url2 + encodeURIComponent(vaultName);
     var queryParameters = [];
-    queryParameters.push('api-version=2014-12-19-PREVIEW');
+    queryParameters.push('api-version=2015-06-01');
```

Comment:
@JeffSimmer Because of this change, you should also update the version in [package.json](https://github.com/Azure/azure-sdk-for-node/blob/master/lib/services/keyVaultManagement/package.json) from "0.10.1" to "0.11.0".

Once the PR is merged, you should be able to publish the package to npm using an internal Jenkins service.